packages/models-library/src/models_library/api_schemas_catalog/services.py

```diff
@@ -205,10 +205,7 @@ class ServiceUpdate(ServiceMetaDataEditable, ServiceAccessRights):
 class ServiceGet(
     ServiceMetaDataPublished, ServiceAccessRights, ServiceMetaDataEditable
 ):  # pylint: disable=too-many-ancestors
-    owner: LowerCaseEmailStr | None = Field(
-        None,
-        description="None when the owner email cannot be found in the database",
```

Comment:
MINOR: keep the description or as comment. I think it is informative of why None is used for